src/main/resources/org/damap/base/db/changeLog-4.x/changeLog-4.x.yaml

```diff
@@ -0,0 +1,7 @@
+databaseChangeLog:
```

Comment:
I do not think further splitting of the changesets is strictly necessary, since there number will always be relatively small. Though, i do not have anything against it either. If you want to keep it, you should write a sentence or two in our documentation of the liquibase workflow. You could also add a sentence about always writing a rollback statement, when necessary.